elixir/apps/web/lib/web/live/resources/show.ex

```diff
@@ -39,7 +39,7 @@ defmodule Web.Resources.Show do
 
   def render(assigns) do
     ~H"""
-    <.breadcrumbs home_path={~p"/#{@account}/dashboard"}>
```

Comment:
Instead of doing this, we should just replace `home_path=` with `account=` attr, and then define what is the home path in the component itself. We have a todo in a `<. breadcrumb />` for that.

Reply:
Fixed